test/core/security/roleRepository.test.js

```diff
@@ -591,22 +595,22 @@ describe('Test: security/roleRepository', () => {
       kuzzle.pluginsManager.plugins = { plugin_test };
     });
 
-    it('should skip non-plugins or wildcarded controllers', () => {
+    it('should skip non-plugins or wildcarded controllers', async () => {
       kuzzle.pluginsManager.isController.returns(false);
 
       const role = new Role();
       role.controllers = { '*': 123 };
 
       kuzzle.funnel.isNativeController.returns(false);
-      should(() => roleRepository.checkRolePluginsRights(role)).not.throw();
+      await should(async () => await roleRepository.checkRolePluginsRights(role)).not.throw();
```

Comment:
async/await methods don't throw, they reject their promises

So it's simpler to write:

```js
      await should(roleRepository.checkRolePluginsRights(role)).be.fulfilled();
```

Or better yet: 

```suggestion
      await roleRepository.checkRolePluginsRights(role);
```

Reply:
Yes you're right :sweat_smile: I just adapted the test quickly without thinking a lot about it..